internal/config/config.go

```diff
@@ -32,10 +32,15 @@ type Config struct {
 	Cpu        uint32   `env:"JAIL_CPU" envDefault:"100"`
 	Pow        uint32   `env:"JAIL_POW"`
 	Port       uint32   `env:"JAIL_PORT" envDefault:"5000"`
+	Dev        []string `env:"JAIL_DEV" envDefault:"null,zero,urandom"`
 	Syscalls   []string `env:"JAIL_SYSCALLS"`
 	TmpSize    size     `env:"JAIL_TMP_SIZE"`
 }
 
+func (c *Config) Proxy() bool {
```

Comment:
This should probably be named `ShouldProxy` or similar (also, in the future split refactors into separate commit pls? :pray:)